packages/ods/src/components/combobox/src/components/ods-combobox/ods-combobox.tsx

```diff
@@ -601,24 +627,52 @@ export class OdsCombobox implements OdsFormElement {
   }
 
   private async openDropdown(): Promise<void> {
-    if (this.resultListElement && this.searchElement) {
+    if (!this.resultListElement || !this.searchElement) {
+      return;
+    }
+
+    this.resultListElement.classList.remove('ods-combobox__results--bottom', 'ods-combobox__results--top');
```

Comment:
should not be duplicated, idealy it should be done on close, and we should always close and reopen, but it's TBD as mentionned on the state comment.